content/blog/vercel-partner-marketplace.mdx

```diff
@@ -0,0 +1,73 @@
+
+### Realtime updates
+
+Use Inngest's [Realtime feature](/docs/features/realtime?ref=blog-vercel-marketplace) to stream data from background workflows to the browser, enabling interactive applications and live status updates. Easily integrate it using our [our React hooks and Next.js examples](/docs/features/realtime/react-hooks?ref=blog-vercel-marketplace).
+
+### Branch environments
+
+Inngest automatically creates a new, isolated environment for every Vercel preview deployment, giving each team member a completely isolated full-stack test environment for every pull request. Branch environments are created instantly and on-demand whenever you deploy your Vercel project. [Learn more about branch environments](/docs/platform/environments?ref=blog-vercel-marketplace#branch-environments).
+
+## Setting up Inngest
+
+If you don't already have an Inngest account, head to [the Inngest page in Vercel Marketplace](https://vercel.com/marketplace/inngest). Click “Install” and agree to the terms and conditions.
+
+![Inngest in the Vercel Marketplace](/assets/blog/vercel-partner-marketplace/vercel-marketplace-1.png)
+
+Next you'll be able to select the billing plan and the included usage for each plan (more info on our plans below). Here we've selected the Pro plan which starts at $75 per month with additional charges for executions above the included amount.
+
+![Selecting a billing plan](/assets/blog/vercel-partner-marketplace/vercel-marketplace-2.png)
+
+On the next screen, you'll give your project a name. We suggest something that matches your project or company. Then click “Continue” to complete your Inngest account setup!
+
+Now that you've created your Inngest account, it's time to connect Inngest to one of your Vercel projects. Here you'll find useful “Getting started” information including how to connect Inngest to a Vercel project as well as set up Inngest in your codebase.
+
+To connect a project, you can either select the “Projects” tab or click the “Connect Project” button on the getting started guide section like in the video below.
+
+<AutoplayVideo src={"https://cdn.inngest.com/blog/vercel-partner-marketplace/vercel-connect-project.mp4"} />
+
+Follow the rest of the Getting started guide and when you next deploy your project to Vercel, your [app will automatically sync with Inngest](/docs/platform/manage/apps?ref=blog-vercel-marketplace) and functions will be ready to be triggered! You can always open the Inngest dashboard by clicking the “**Open in Inngest**” button at the top right.
+
+![Inngest dashboard](/assets/blog/vercel-partner-marketplace/inngest-dashboard.png)
+
+## Unified billing through Vercel
```

Comment:
and usage metrics/observability*

Reply:
I think the main value is unified billing and usage metrics are mostly just to enable the billing, so I think it's OK to keep simpler with less text.